### ISSUE TITLE
Diego week ticket

### ISSUE THREAD

**Issue by Diegokrenz** [BradleyBartelt/P2__Waves#18]
Added contact us page:
runtime link: http://www.site.pieceofthepi.cf/contact
link to code:
https://github.com/BradleyBartelt/P2__Waves/blob/eb00c688fa59e469ad8853afda4d555e424e0f11/templates/contact.html#L4

**Comment by BradleyBartelt** [comment]
good idea, most sites have contact pages, however, it would be intresting if you added some logic some how.

**Comment by ghost** [comment]
Its quite to see our website to be more fleshed out. You could definitely add in more algorithms to make the display more appealing.

**Comment by colin-szeto** [comment]
Look forward to see you use jinja to iterate through a dictionary to display information rather than typing in everything manually. Will work with you on implementing jinja
